static/includes/AppsUnversioned.md

```diff
@@ -1,6 +1,8 @@
+&NewLine;
+
 Application maintenance is independent from TrueNAS SCALE version release cycles.
-App versions, features, fields, and install behavior at the time of access might vary from those in documented tutorials, especially for legacy version documentation that no longer receives active updates.
+This means app version information, features, configuration options, and installation behavior at the time of access might vary from those in documented tutorials.
 
-We welcome community contributions to keep this documentation current!
-Click **Edit Page** in the top right corner to propose changes to this article.
-See [Updating Content](https://www.truenas.com/docs/contributing/documentation/contentupdate/) for more information.
+We welcome community contributions to keep our documentation current!
+Click **Edit Page** in the top right corner to propose changes to an article.
+See [Updating Content](https://www.truenas.com/docs/contributing/documentation/contentupdate/) for more information.
```

Comment:
Would this generic text applied to 24.04 apps tutorials lead a contributor to try updating 24.04 tutorials when we'd rather see their changes go into the master branch content instead? What about focusing contributors on the eventual single location for apps content to avoid good content updates getting merged to versioned/EoL docs?

 ```suggestion
We welcome community contributions to keep our documentation current!
Switch to the TrueNAS SCALE Nightly version of this content and click **Edit Page** in the top right corner to propose changes to an article.
See [Updating Content](https://www.truenas.com/docs/contributing/documentation/contentupdate/) for more information.
```

Reply:
I think this was a change suggested by Dan to make the contribution statement more "global" that I accepted.  I originally had more specific instructions related to 24.04 and Master. 24.04 is still the active release so we should allow users to make changes to those tutorials in that branch while we work on getting the official app tutorials in Master updated. We are not going to go back to 24.04 to update that content ourselves. All this said, the proposed change is acceptable.